.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,64 @@
+name: SonarCloud
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  info:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Dump GitHub context
+      env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+      run: echo "$GITHUB_CONTEXT"
+
+  static_analysis:
+    runs-on: ubuntu-latest
+
+    container:
+        image: navitia/debian10_dev
```

Comment:
our platform of production is running on debian8(the migration towards debian10 is in progress), maybe we should do it on debian8 too?

Reply:
I understand, we can try to move to debian8. 
I'm just afraid about the gcovr version under debian8, I need at least the version 4.2.

Reply:
OK, i understand better why you choose to use debian10... and if it's only for the code and no the runtime, I'm ok to stay on debian10